apps/examples/nettest/nettest.c

```diff
@@ -650,67 +731,83 @@ int main(int argc, FAR char *argv[])
 int nettest_main(int argc, char *argv[])
 #endif
 {
-	nlldbg("Running nettest_main \n");
-	int proto = 0;
-	int num_packets_to_process = NUM_PACKETS;
-	uint32_t sleep_time = 0;
+	int mode = 0;
+	nettest_proto_e proto = 0;
```

Comment:
Does this line mean that the default value of proto is NT_TCP?

Reply:
I will add an addtional field.